core/module/define_method_spec.rb

```diff
@@ -115,6 +115,23 @@ def test_method
 end
 
 describe "Module#define_method" do
+  it "does not call method_added on a BasicObject singleton class" do
+    object = BasicObject
+    def_target = class << object; self; end
+
+    method_added_messages = []
+
+    def_target.class_exec do
+      define_method(:method_added) { |name| method_added_messages << "howdy #{name}" }
+    end
+
+    def_target.class_exec do
+      define_method(:method_added) { |*args| }
+    end
+
+    method_added_messages.should == []
+  end
+
```

Comment:
Could you move it after `it "calls #method_added after the method is added to the Module" do`?